pkgs/development/python-modules/json-rpc/default.nix

```diff
@@ -0,0 +1,22 @@
+{ lib, isPy27, buildPythonPackage, fetchPypi, pytestCheckHook, mock }:
+
+buildPythonPackage rec {
+  pname = "json-rpc";
+  version = "1.13.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "12bmblnznk174hqg2irggx4hd3cq1nczbwkpsqqzr13hbg7xpw6y";
+  };
+
+  checkInputs = [ pytestCheckHook ];
+
+  nativeBuildInputs = lib.lists.optional isPy27 mock;
```

Comment:
```suggestion
  nativeBuildInputs = lib.optional isPy27 mock;
```

Reply:
Honestly, I am not sure. There are at least some references to `mock` for certain Python versions in the tests. Thus, I added `mock` for the checks. Thanks for noticing.